src/NuGet.Commands/RestoreCommand.cs

```diff
@@ -42,14 +43,14 @@ public RestoreCommand(ILoggerFactory loggerFactory)
                 return new RestoreResult(success: false, restoreGraphs: Enumerable.Empty<RestoreTargetGraph>());
             }
 
-            var projectLockFilePath = Path.Combine(request.Project.BaseDirectory, LockFileFormat.LockFileName);
+            var projectLockFilePath = string.IsNullOrEmpty(request.LockFilePath) ?
```

Comment:
A lot of the real benefits in `dnu restore` was primarily from restoring multiple project.json in parallel. We should figure that out as a secondary work item.